### DISCUSSION TITLE
OLED Display angezeigte Informationen

### DISCUSSION THREAD

**Discussion by frankh-gituser** [hoylabs/OpenDTU-OnBattery#554] [Ideas]
Hallo, meine beiden DTUs habe ich jetzt auch jeweils mit einem  OLED Display erweitert. Beeindruckendes Kosten/Nutzen Verhältnis. In der Grundeinstellung werden jetzt die Info von dem Inverter angezeigt. Meine Frage wäre, ob für die Zukunft auch geplant ist, die Auswahl der angezeigten Informationen zu erweitern, zB auf die Victron Ladeleistung etc ? Die Anzeige der Informationen könnte ja zB durch Wechsel der angezeigten Quelle umgesetzt werden. Im Sourcecode scheint einiges der Umsetzung in Display_Graphic zu erfolgen. Aber vielleicht ist so eine Erweiterung auch in einem der nächsten Releases schon geplant.  Danke vorab

**Comment by spcqike** [comment]
es ist relativ einfach, sich die Software für seine Bedürfnisse anzupassen. ich habe die OLED Ausgabe bspw so angepasst, dass ich im 5-Sekundentakt die PV Produktion und den Netzbezug sehe (ich habe keinen Speicher, daher ist für mich ein "was produziere ich grade und wie viel speise ich ggf. ein?" am interessantesten.

https://github.com/spcqike/OpenDTU-OnBattery/blob/f11727dee8702dd11031b9563eb88c57e2416101/src/Display_Graphic.cpp#L151-L167

eine frei anpassbare Anzeige wäre zwar schön, aber eben auch komplex in der Umsetzung. Vordefinierte Szenarien wären wohl einfacher, aber welche Szenarien will man da abdecken?

**Reply by frankh-gituser** [reply]
Ich habe zwar Speicher an den DTUs, aber deine Lösung entspricht genau dem. was ich suche. An dem kleinen Display brauche ich primär, was vom Victron reinkommt und was über den Inverter eingespeist wird. Schaue ich mir an. Danke für die Rückmeldung.
